cmd/keytransparency-client/cmd/post.go

```diff
@@ -20,6 +20,7 @@ import (
 	"log"
 	"time"
 
+	"github.com/google/keytransparency/core/authentication"
```

Comment:
For consistency, please add a new line here.  We're organizing our imports as follows:

```go
import(
    "golang packages"
   
     "keytransparency packages"

     "other packages"
)
```